.travis.yml

```diff
@@ -18,9 +18,13 @@ cache:
     - /home/travis/.cargo
 before_script:
   - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
+  - rustup component add rustfmt
 script:
   - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then cargo clippy --all-targets --all-features -- -D warnings; fi
+  - cargo fmt --all --write-mode diff
```

Comment:
What's the difference between `--write-mode diff` and `--check`? `rustfmt`'s [instructions for CI](https://github.com/rust-lang/rustfmt#checking-style-on-a-ci-server) seem to prefer `--check`.

Reply:
yup my bad. I misread something.